dart/deepgram-Transcribe-Audio/lib/main.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+void returnSuccess(final res, final data) {
+  res.json({
+    'success': true,
+    'deepgramData': data,
+  });
+}
+
+void returnFailure(final res, final message) {
+  res.json({
+    'success': false,
+    'message': message,
+  });
+}
+
+Future<void> start(final req, final res) async {
+  var fileUrl;
+  var deepgramApiKey;
+
+  //getting data from payload
+  try {
+    final data = jsonDecode(req.payload);
+    fileUrl = data["fileUrl"];
+    deepgramApiKey = req.env["DEEPGRAM_API_KEY"];
```

Comment:
In the newer version of functions, the variables are in `req.variables` rather than `req.env`:

```suggestion
    deepgramApiKey = req.variables["DEEPGRAM_API_KEY"];
```